migration/idempotent/001-base.sql

```diff
@@ -3384,73 +3384,203 @@ $$
 LANGUAGE PLPGSQL STABLE;
 GRANT EXECUTE ON FUNCTION _prom_catalog.get_metrics_that_need_compression() TO prom_maintenance;
```

Comment:
Shouldn't we get rid of it?

Reply:
let me see if i can tell if it's used anywhere else...

Reply:
removing...